RecoTracker/MkFit/python/customizeInitialStepOnly.py

```diff
@@ -0,0 +1,146 @@
+    process.initialStepSeedLayers.FPix.HitProducer = 'siPixelRecHitsPreSplitting'
+    process.initialStepSeedLayers.BPix.HitProducer = 'siPixelRecHitsPreSplitting'
+    process.initialStepHitQuadruplets.SeedComparitorPSet.clusterShapeCacheSrc = "siPixelClusterShapeCachePreSplitting"
+    process.initialStepSeeds.SeedComparitorPSet.ClusterShapeCacheSrc = "siPixelClusterShapeCachePreSplitting"
+    if hasattr(process.initialStepTrackCandidates, "measurementTrackerEvent"):
+        # MkFit case
+        process.initialStepTrackCandidates.measurementTrackerEvent = 'MeasurementTrackerEventPreSplitting'
+        process.initialStepTrackCandidatesMkFitInput.pixelRecHits = "siPixelRecHitsPreSplitting"
+    else:
+        process.initialStepTrackCandidates.MeasurementTrackerEvent = 'MeasurementTrackerEventPreSplitting'
+    process.initialStepTracks.MeasurementTrackerEvent = 'MeasurementTrackerEventPreSplitting'
+    process.iterTrackingTask = cms.Task(process.trackerClusterCheck,
+                                        process.InitialStepTask)
+
+    # Customize MTV
+    for selector in [
+        process.cutsRecoTracksInitialStep,
+        process.cutsRecoTracksPt09InitialStep,
+        process.cutsRecoTracksFromPVInitialStep,
+        process.cutsRecoTracksFromPVPt09InitialStep,
+        ]:
+        selector.algorithm = []
+        selector.quality = []
+        selector.src = "initialStepTracks"
+        selector.vertexTag = "firstStepPrimaryVertices"
+
+    process.trackingParticleRecoTrackAsssociationPreSplitting = process.trackingParticleRecoTrackAsssociation.clone(
+        label_tr = "initialStepTracks",
+        associator = "quickTrackAssociatorByHitsPreSplitting",
+    )
+    process.VertexAssociatorByPositionAndTracksPreSplitting = process.VertexAssociatorByPositionAndTracks.clone(
+        trackAssociation = "trackingParticleRecoTrackAsssociationPreSplitting"
+    )
+
+
+    def setInput(mtvs, labels):
+        for mtv in mtvs:
+            mod = getattr(process, mtv)
+            mod.label = labels
+            mod.label_vertex = "firstStepPrimaryVertices"
+            if mod.UseAssociators.value():
+                mod.associators = ["quickTrackAssociatorByHitsPreSplitting"]
+            else:
+                mod.associators = ["trackingParticleRecoTrackAsssociationPreSplitting"]
+            mod.vertexAssociator = "VertexAssociatorByPositionAndTracksPreSplitting"
+            mod.trackCollectionForDrCalculation = "initialStepTracks"
+            mod.dodEdxPlots = False
+            mod.doResolutionPlotsForLabels = []
+
+    setInput(["trackValidatorTrackingOnly", "trackValidatorAllTPEfficStandalone", "trackValidatorTPPtLess09Standalone", "trackValidatorBHadronTrackingOnly"],
+             ["cutsRecoTracksInitialStep", "cutsRecoTracksPt09InitialStep"])
+    setInput(["trackValidatorFromPVStandalone", "trackValidatorFromPVAllTPStandalone"], ["cutsRecoTracksFromPVInitialStep", "cutsRecoTracksFromPVPt09InitialStep"])
```

Comment:
some linebreaks would be nice for readability on github

Reply:
Will add.